[2020-12-09 00:25:24,613] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:50:00+00:00 [queued]>
[2020-12-09 00:25:24,628] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:50:00+00:00 [queued]>
[2020-12-09 00:25:24,628] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2020-12-09 00:25:24,628] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2020-12-09 00:25:24,629] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2020-12-09 00:25:24,636] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): Data_from_table2> on 2020-12-08T18:50:00+00:00
[2020-12-09 00:25:24,639] {standard_task_runner.py:54} INFO - Started process 28016 to run task
[2020-12-09 00:25:24,673] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'Airflow_task', 'Data_from_table2', '2020-12-08T18:50:00+00:00', '--job_id', '336', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/server-elastic.py', '--cfg_path', '/var/folders/x0/npyjyyg15zj3krdkh0pv612w0000gn/T/tmp9l6zp231']
[2020-12-09 00:25:24,675] {standard_task_runner.py:78} INFO - Job 336: Subtask Data_from_table2
[2020-12-09 00:25:24,715] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: Airflow_task.Data_from_table2 2020-12-08T18:50:00+00:00 [running]> 153.225.168.192.in-addr.arpa
[2020-12-09 00:25:29,185] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-09 00:25:29,187] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2020-12-09 00:25:29,188] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-09 00:25:29,189] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:25:29,234] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: connecting to localhost:9092 [('::1', 9092, 0, 0) IPv6]
[2020-12-09 00:25:29,234] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]> returned error 61. Disconnecting.
[2020-12-09 00:25:29,234] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv6 ('::1', 9092, 0, 0)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:25:29,284] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: connecting to localhost:9092 [('127.0.0.1', 9092) IPv4]
[2020-12-09 00:25:29,284] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]> returned error 61. Disconnecting.
[2020-12-09 00:25:29,284] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=localhost:9092 <connecting> [IPv4 ('127.0.0.1', 9092)]>: Closing connection. KafkaConnectionError: 61 ECONNREFUSED
[2020-12-09 00:25:29,285] {logging_mixin.py:112} INFO - Error::::: NoBrokersAvailable
[2020-12-09 00:25:29,285] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-12-09 00:25:29,290] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=Airflow_task, task_id=Data_from_table2, execution_date=20201208T185000, start_date=20201208T185524, end_date=20201208T185529
[2020-12-09 00:25:29,600] {local_task_job.py:102} INFO - Task exited with return code 0
